fem/coefficient.cpp

```diff
@@ -775,4 +775,93 @@ double ComputeGlobalLpNorm(double p, VectorCoefficient &coeff, ParMesh &pmesh,
 }
 #endif
 
+VectorQuadratureFunctionCoefficient::VectorQuadratureFunctionCoefficient(
+   QuadratureFunction *qf)
+   : VectorCoefficient(qf->GetVDim()), QuadF(qf), index(0),
```

Comment:
Since these constructors would segfault if `qf` is null, you should be giving it as a reference instead of a pointer. Also, the class member should be a `const` reference, as null doesn't seem like a valid value.

Reply:
Good suggestion changed this over to a reference and the class member a const reference.